topics/statistics/tutorials/RNN/tutorial.md

```diff
@@ -0,0 +1,364 @@
+---
+layout: tutorial_hands_on
+
+title: Introduction to recurrent neural networks (RNN)
+zenodo_link: https://zenodo.org/record/4477881
+questions:
+- What is a recurrent neural network (RNN)?
+- What are some applications of RNN?
+objectives:
+- Understand the difference between feedforward neural networks (FNN) and RNN
+- Learn various RNN types and architectures
+- Solve a sentiment analysis problem on IMDB movie review dataset using RNN in Galaxy
+requirements:
+  -
+    type: internal
+    topic_name: statistics
+    tutorials:
+      - intro_deep_learning
+time_estimation: 2H
+contributors:
+- kxk302
+
+---
+
+
+
+
+# Introduction
+{:.no_toc}
+
+<!-- This is a comment. -->
+
+Artificial neural networks are a machine learning discipline roughly inspired by how neurons in a 
+human brain work. In the past decade, there has been a huge resurgence of neural networks thanks 
+to the vast availability of training data and enormous increases in computing capacity (Successfully 
+training complex neural networks in some domains requires lots of data and compute capacity). There 
+are various types of neural networks (Feedforward, recurrent, etc). In this tutorial, we discuss 
+the recurrent neural networks and explain how they differ from feedforward variants. We also describe 
+various RNN architectures and solve a sentiment analysis problem using RNN in Galaxy.       
```

Comment:
An explanation should be there why RNN is used for sentiment analysis problem (because RNN is generally used for temporal data such as text, speech and so on ...). Please also add some successful examples (links to papers) where RNN is used for analysis of temporal data to provide more context to readers. Thanks.

Reply:
Will do.